src/Components/dashboard/route/Skeletons/Skeleton.css

```diff
@@ -0,0 +1,96 @@
+/* basic styles */
+
+.skeleton {
+    background-color: rgb(207, 207, 207);
+    margin: 10px 0;
+    border-radius: 4px; 
+}
+
+.skeleton.addroute{
+    width: 100%;
+    height: 12px;
+    background-color: rgb(207, 207, 207);
+}
+.skeleton.title {
+    width: 50%;
+    height: 20px;
+    margin-bottom: 15px;
+}
+.skeleton.avatar{
+    width: 100px;
+    height: 100px;
+    border-radius: 50%;
+}
+.skeleton.thumbnail{
+    width: 100px;
+    height: 100px;
+}
+
+
+.card-skeleton{
+    display:flex;
+    justify-content: center;
+    align-items:center;
+    width:53%;
+    height:100%;
+    padding:.5rem;
+    margin: 1rem;
+    border:1px solid grey;
+}
+
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+>>>>>>> 38aedc10... ft-routes-crud
+
+
+.card-skeleton .main-col{    
+    flex: 1rem;    
+}
+
+<<<<<<< HEAD
+=======
+>>>>>>> a7a64daa... (ft-routes-crud)
+=======
+>>>>>>> 38aedc10... ft-routes-crud
+.card-skeleton .middle-col{
+    flex: 1rem;
+    margin-right: 1rem;   
+    height: 60%;
+    width: 100%;
+}
+
+.card-skeleton .right-col{
+    flex: 1rem;
+    margin-right: 1rem; 
+    height: 60%; 
+    width: 100%;
+}
+
+
+.card-skeleton .left-col{
+    margin-right: 1rem;
+    flex: 1rem;
+    height: 60%;
+    width: 100%;
+<<<<<<< HEAD
```

Comment:
Same here, resolve these conflicts.

Reply:
Look like I missed this ones, I have resolved them